common-extensions/src/main/java/com/twocoders/util/commonextensions/CollectionExtensions.kt

```diff
@@ -0,0 +1,27 @@
+package com.twocoders.util.commonextensions
+
+import android.util.SparseArray
+
+val SparseArray<*>.max: Int get() = size() - 1
```

Comment:
To max by som bez dokumentacie alebo kodu povazoval za najdenid najvacsieho cisla. A pre for cyklus mi to pride zbytocne, kedze pouzitim `until size()` dosiahnes to iste